assets/i18n/ar.json

```diff
@@ -6,7 +6,7 @@
 	"I18N_ABOUT_PAGE_ABOUT_TAB_HEADING": "حول أوبيا",
 	"I18N_ABOUT_PAGE_ABOUT_TAB_PARAGRAPH_1": "مهمة أوبيا هي مساعدة أي شخص على تعلم أي شيء يريده بطريقة فعالة وممتعة.",
 	"I18N_ABOUT_PAGE_ABOUT_TAB_PARAGRAPH_2": "من خلال إنشاء مجموعة من الدروس المجانية، وعالية الجودة، وواضحة بشكل فعال بمساعدة من المربين من جميع أنحاء العالم؛يهدف أوبيا لتزويد الطلاب بتعليم جيد &mdash; بغض النظر عن مكان وجودهم أو الموارد التقليدية التي يمكنهم الوصول إليها.",
-	"I18N_ABOUT_PAGE_ABOUT_TAB_PARAGRAPH_3": "حتى الآن، أنشأ اختصاصيو التوعية أكثر من <[numberOfExplorations]> من هذه الدروس، التي نسميها <em>explorations</em>، ويخدمون تقريبا <[numberofStudentsServed]> طالب في جميع أنحاء العالم.",
+	"I18N_ABOUT_PAGE_ABOUT_TAB_PARAGRAPH_3": "حتى الآن، أنشأ اختصاصيو التوعية أكثر من {{ numberOfExplorations }} من هذه الدروس، التي نسميها <em>explorations</em>، ويخدمون تقريبا {{ numberofStudentsServed}} طالب في جميع أنحاء العالم.",
```

Comment:
This will be done incrementally. Angular uses `{{ }}` for string interpolation and AngularJS uses `<[]>`. So, whatever directive we migrate to Angular uses i18n convention, we need to make the corresponding changes here in all the json files.

Reply:
Hmmm does everyone know about the fact that Angular will be using `{{}}` I'm fine with it, since using `{{}}` is a convention (even in AngularJS) and we used `<[]>` mostly because we used backend interpolation with Jinja before. But this should be explained and passed through our community so everyone understands this.

cc @seanlip @DubeySandeep

Reply:
Do not do this. This sort of thing needs to be done carefully and in stages. Here is the ordering of steps:

1. Confirm with @vojtechjelinek that Jinja has been removed completely and that there are no more usages of {{...}} in the codebase. If not, remove Jinja completely first and get rid of all those usages.

2. Change the interpolation in AngularJS to {{...}} instead of `<[...]>`.

3. Migrate without changing interpolation.

Reply:
I also suspect that, in the near future at least, we will still be serving pages using Jinja. So you might also consider changing the interpolation symbols: https://stackoverflow.com/questions/49211263/change-interpolation-symbols-in-angular5

Reply:
We don't use Jinja for around half a year and I don!t think there is any need to use it. 

> I also suspect that, in the near future at least, we will still be serving pages using Jinja 

@seanlip Why?

Reply:
Wait, maybe I don't understand something. I searched in the codebase for "jinja" and things like jinja_utils and get_jinja_env are still there. And it's still on core/controllers/base.py. So aren't we still using it?

Reply:
We use it for serving files that are not interpolated in any way. In the speed team, we found out that we have different priorities than moving to our custom static serving so we use Jinja without any interpolated values.

Reply:
Right -- but that still means that if we use {{...}} in those files, Jinja may still interpret them as containing stuff to be interpolated, right?

Reply:
@seanlip, @vojtechjelinek I was going through the codebase, there are outdated files in folder `backend_prod_files`. I checked lines 53-55 in feconf.py which mention these.
```
# backend_prod_files contain processed JS and HTML files that are served by
# Jinja, we are moving away from Jinja so this folder might not be needed later
# (#6964)
```
It is written it might not be needed later, isn't there any sort of tracker for this?

Also in the changelog line 773:
`Omit backend_prod_files during linting (#7533)`
Not just this, there are a lot for prs mentioning about replacing some Jinja templates.

Not really sure but maybe Jinja has been removed because all the Jinja templates seem to reside in `backend_prod_files` folder and they are pretty outdated.

Reply:
@seanlip @vojtechjelinek  Never mind my last comment, hadn't refreshed my page before posting.

Reply:
@seanlip Oh yep, that is true, sorry didn't realize this.

Reply:
@vojtechjelinek Can we actually get rid of it? How hard would it be (what are the steps)?

Reply:
> Right -- but that still means that if we use {{...}} in those files, Jinja may still interpret them as containing stuff to be interpolated, right?

@seanlip will it though? These are in json files. I don't think Jinja interpolates json files.

As for the angular HTML files, when angular2-template-loader compiles it, it moves the HTML content into the bundle.js file. Attaching a screenshot of the same.
![image](https://user-images.githubusercontent.com/23002114/81495885-2bddd580-92d1-11ea-93f1-c3624e270aef.png)

As you can see the about-page.component.html content is now present in about.bundle.js. I think we can use `{{...}}`.

Reply:
(Reposting to put my comment in the right place.)

We talked about this already. There is the aspect of developer confusion which we wish to avoid. I understand that this PR may not introduce issues with {{...}} but there still needs to be a very clear separation between jinja stuff and Angular stuff, for all the reasons previously stated, as long as we still use jinja.

Reply:
@seanlip, @vojtechjelinek please correct me if I am wrong.
For serving the HTML files, I think it as simple as changing lines 379-380 in base.py in function `render_template` from
```
self.response.write(
  self.jinja2_env.get_template(filepath).render(**values))
```
to this
```
html_file = open(filepath + filename).read()
self.response.write(html_file)
```
where filepath is the path to the webpack_bundles folder.

Reply:
Hmm, it would need handling of prod vs dev mode, and we also wanted to add some kind of caching.